crates/core/src/registry.rs

```diff
@@ -198,18 +199,20 @@ pub async fn get_all_versions_descending(dependency_name: &str) -> Result<Versio
 
     match revision
         .data
-        .iter()
+        .par_iter()
```

Comment:
This parallel iterator is about 10x slower than a normal iterator:

![image](https://github.com/user-attachments/assets/6a753e8a-8fa9-4084-8e50-4cce4a098ed3)